runtime/src/constants.rs

```diff
@@ -31,7 +31,7 @@ pub mod currency {
     pub const EXISTENTIAL_DEPOSIT: Balance = 0;
```

Comment:
Makes sense to set this as zkVerify ?

Reply:
That's a good question. The horizen's guys choose this approach to be etherum compatible (like moonbeen), but we can choose a different approach.